tourney/commands/join_command.py

```diff
@@ -12,10 +14,46 @@ def execute(self, lookup=None):
     participants = state.participants()
     user_name = lookup.user_name_by_id(self.user_id())
 
-    if self.user_id() not in participants:
-      state.add_participant(self.user_id())
-      state.save()
-      Achievements.get().interact(JoinBehavior(self.user_id()))
-      return "{}, you've joined today's game!".format(user_name)
+    teams = state.teams()
+    teamNames = state.team_names()
 
-    return "{}, you've _already_ joined today's game!".format(user_name)
+    created_teams = len(teams) > 0
+    if created_teams:
+      matches = state.unrecorded_matches()
+      teams_with_matches = []
+      for match in matches:
```

Comment:
As we talked about, it would be great to use `Scores.today()` to exclude matches that have already been played. Note that it doesn't return team indices but the members of the teams, so a reverse-translation needs to take place.